src/components/play-page.js

```diff
@@ -165,9 +166,11 @@ export class FlashCards extends connect(store)(PageViewElement) {
         return;
       }
       const activeCardData = this._cards[activeCard.hint][activeCard.index];
+      var answers = [].concat.apply([], [activeCardData.en]);
```

Comment:
pls use `const` instead of `var`

Reply:
also i thiiiiiiiiiiiiink this can just be `[].concat(activeCardData.en)`. `concat` works both with arrays and elements:
```
[1,2,3].concat([4,5])
>  [1, 2, 3, 4, 5]
[1,2,3].concat(4)
>  [1, 2, 3, 4]
```